### PR TITLE
Add basic rtest.xml

### PR DESCRIPTION
**Pull request by lawruble13**
Add rtest.xml with basic sets. Required by devops.

### CODE REVIEW

**Review by YvanMokwinski — APPROVED** [review]

**Review by ntrost57 — APPROVED** [review]

File: rtest.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE testset SYSTEM "/usr/local/share/rtest.dtd">
```
Comment (saadrahim) [inline rtest.xml:2]: Is this hardcoding correct?

Comment (TorreZuk) [reply rtest.xml:2]: That doesn't look windows friendly.   
You could also leave the dtd specification out of the rtest.xml for now and use a dtd for only an external validation process.

Comment (lawruble13) [reply rtest.xml:2]: For the moment, the DTD isn't used for validation of `rtest.xml`, and this doctype declaration was primarily used for my IDE and can safely be removed. Eventually we'd like to have the dtd at a static location and reference using a public doctype.

**Comment by lawruble13** [comment]
Basic rtest.xml added in #197 .

**Pull request closed without merging**